src/main/java/ch/njol/skript/util/Timespan.java

```diff
@@ -248,66 +185,59 @@ public Timespan() {
 	 * @param millis The milliseconds of Timespan
 	 */
 	public Timespan(long millis) {
-		if (millis < 0)
-			throw new IllegalArgumentException("millis must be >= 0");
+		Preconditions.checkArgument(millis >= 0, "millis must be >= 0");
 		this.millis = millis;
 	}
 
 	/**
 	 * Builds a Timespan from the given long parameter of a specific {@link TimePeriod}.
 	 *
 	 * @param timePeriod The requested TimePeriod
-	 * @param time The time of the requested TimePeriod
+	 * @param time       The time of the requested TimePeriod
```

Comment:
?

Reply:
intellij's auto-format makes it line up with the other params